test/PushNotificationsTests/APITests.cpp

```diff
@@ -0,0 +1,104 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+#include "pch.h"
+#include <testdef.h>
+#include "Shared.h"
+
+using namespace WEX::Common;
+using namespace WEX::Logging;
+using namespace WEX::TestExecution;
+
+using namespace winrt;
+using namespace winrt::Microsoft::ApplicationModel::Activation;
+using namespace winrt::Microsoft::ProjectReunion;
+using namespace winrt::Windows::ApplicationModel;
+using namespace winrt::Windows::ApplicationModel::Activation;
+using namespace winrt::Windows::Foundation;
+using namespace winrt::Windows::Foundation::Collections;
+using namespace winrt::Windows::Management::Deployment;
+using namespace winrt::Windows::Storage;
+using namespace winrt::Windows::System;
+
+// TODO: Write Register/Unregister tests that utilize the Assoc APIs to validate results.
+
+namespace Test::PushNotifications
+{
+    class APITests
+    {
+    private:
+        wil::unique_event m_failed;
+
+        const std::wstring c_testPackageFile = g_deploymentDir + L"MSIXPackager_1.0.0.0_x86_x64_Debug.msixbundle";
+        const std::wstring c_testPackageCertFile = g_deploymentDir + L"MSIXPackager_1.0.0.0_x86_x64_Debug.cer";
+        const std::wstring c_testPackageFullName = L"PushNotificationsWin32App_9eg3vg7cmy3rj"; // Need to replace
+
+    public:
+        BEGIN_TEST_CLASS(APITests)
+            TEST_CLASS_PROPERTY(L"Description", L"Project Reunion Push Notifications test")
+            TEST_CLASS_PROPERTY(L"IsolationLevel", L"Method")
+            TEST_CLASS_PROPERTY(L"ThreadingModel", L"MTA")
+            TEST_CLASS_PROPERTY(L"RunFixtureAs:Class", L"InteractiveUser")
```

Comment:
What is the purpose of this TAEF property?

Reply:
We use 'InteractiveUser' so we have the permissions to use RunCertUtil.